### PR TITLE
DSD-652: Card custom image aspect ratio fix

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-652](https://jira.nypl.org/browse/DSD-652)

## This PR does the following:

- Fixes the bottom padding on custom image components/elements passed to the `Card` component by passing the appropriate aspect ratio value.
- Also fixes the logic in the `Image` component so custom components/elements don't get wrapped in cropping and aspect ratio divs.

## How has this been tested?

Locally in Storybook and through the NYPL test app with the Nextjs `Image` component.

## Accessibility concerns or updates

- None.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -178,6 +178,9 @@ export default function Card(props: React.PropsWithChildren<CardProps>) {
     mainActionLink,
   } = props;
   const hasImage = imageSrc || imageComponent;
+  const finalImageAspectRatio = imageComponent
```
Comment (bigfishdesign13) [inline src/components/Card/Card.tsx:184]: Can you add a console warning if both a `imageComponent` and `imageAspectRatio` are set?  The message should say that `imageAspectRatio` will be ignored in favor of the aspect ratio on `imageComponent`.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Looks good.  Just need to add the console warning.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Review by bigfishdesign13 — APPROVED** [review]
Very nice.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
@wluisi can you take a look when you get a chance?

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr818-fgntzdfg9hb5y5bqb39buazzpkhfzs6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c1fff85f46ed13d84f18f7

**Review by nypl-wluisi — APPROVED** [review]
Tested this branch out in the Scout app and it works. I can remove the wrapper box div around the Card images now, and no more extra bottom padding!

**Pull request merged**